subgraphs/compound-forks/Makefile

```diff
@@ -1,6 +1,6 @@
 .PHONY: codegen
 codegen:
-	graph codegen $(subgraph)/subgraph.yaml -o $(subgraph)/generated
+	graph codegen $(subgraph)/subgraph.yaml -o ./generated
```

Comment:
i don't think we want to change this ... we want `make codegen` to generate code in specific protocols, not the root

we only need to run `graph codegen bastion-protocol/subgraph.yaml -o ./generated` once to generate code under root, while we run `make codegen subgraph=xxx` very often when developing a fork